### PR TITLE
Bump brainstem-js to 0.4.25

### PR DESCRIPTION
**Pull request by naomiajacobs**

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -169,6 +184,231 @@
       "from": "babel-generator@>=6.20.0 <7.0.0",
       "resolved": "https://registry.npmjs.org/babel-generator/-/babel-generator-6.20.0.tgz"
     },
+    "babel-helper-call-delegate": {
```
Comment (juanca) [inline npm-shrinkwrap.json]: What version of `npm` was this generated from?

We should all be on:

```
> node -v
v6.10.1
> npm -v
3.10.10
```

Comment (juanca) [inline npm-shrinkwrap.json]: Why I'm asking: it's kinda weird we added these dependencies from a brainstem-js bump?

Comment (naomiajacobs) [inline npm-shrinkwrap.json]: Whoahhhhh yeah this is weird. I will check this tomorrow.

Comment (naganowl) [inline npm-shrinkwrap.json]: We probably should regenerate this file from scratch with the aforementioned versions.

**Comment by naomiajacobs** [comment]
@juanca @naganowl I re-generated the shrinkwrap with the above node and npm versions, it still seems to have knocked out like 90% of our node modules. Specs are red - may have to do with it running on an older version of node and npm. Green locally.

**Comment by juanca** [comment]
Looks like it's missing dev dependencies.

Try re-generating the shrinkwrap with dev dependencies (implies that CI will have stable testing frameworks)

**Comment by naomiajacobs** [comment]
@juanca re-generated with `devDependencies`. Seems like now `dev: true` got added to stuff, but also a bunch of other packages. I'm unclear about if that's a problem or not. Specs are red on CI, green locally.

**Comment by naomiajacobs** [comment]
@juanca @alan I got annoyed with the `shrinkwrap`, so I got rid of it and made a `yarn.lock` file. I also updated our `package.json` scripts and circle config to use it. Circle is green now, so I either did it right or broke stuff so badly that we can't see it. Would love some more eyes on it to make sure it's the former. 😛

File: circle.yml [hunk 2]
```diff
@@ -1,3 +1,16 @@
 machine:
   node:
-    version: 6.5.0
+    version: 6.10.1
+dependencies:
```
Comment (juanca) [inline circle.yml]: I was under the impression that yarn was officially supported by Circle implying that we do not need to do any setup ourselves.

Comment (naomiajacobs) [inline circle.yml]: From the circle docs: I think it supports it officially in that we don't have to worry about yarn being globally installed. https://circleci.com/docs/1.0/yarn/

**Pull request merged**